torch/_export/exported_program.py

```diff
@@ -334,29 +334,29 @@ def _process_constraints(
                     equality_constraints.append((node_dim, other_node_dim))
 
     # Create dict mapping symbol to a singular range (lower, upper)
-    range_constraints: Dict[sympy.Symbol, RangeConstraint] = {}
+    range_constraints: Dict[sympy.Symbol, ValueRanges] = {}
 
     # Add inline constraints to range_constraints
     for symbol, value_range in inline_constraints.items():
-        range_constraints[symbol] = RangeConstraint(value_range.lower, value_range.upper)
+        range_constraints[symbol] = ValueRanges(value_range.lower, value_range.upper)
```

Comment:
pretty sure `value_range`is already of type `ValueRanges`. In which case you can initialize `range_constraints = {**inline_constraints}`